src/clj/fluree/db/api.cljc

```diff
@@ -293,7 +293,7 @@
   ([ds cred-query {:keys [default-allow? values-map] :as opts}]
    (promise-wrap
     (go-try
-     (let [{query :subject, identity :did} (<? (cred/verify cred-query))]
+     (let [{query :subject, identity :did} (<? (cred/verify cred-query opts))]
```

Comment:
Per the many comments in https://github.com/fluree/db/pull/843, nobody wanted to create additional options to `cred/verify` - so most of the commits after the first one were to take an alternative approach.

This will need to be refactored to take the same (or different) approach.